concepticondata/concepticon.tsv

```diff
@@ -3142,3 +3142,11 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3146	CHEETAH	Animals	Wild Animal of the cat family with characteristic dark spots.	Person/Thing	
 3147	HAMSTER	Animals	Mouse-sized animal with large cheeks for storing food, often kept as a pet.	Person/Thing	
 3148	MEERKAT	Animals	Southern African animal known for its long tail and often seen standing up on its back legs.	Person/Thing	
+3149	ARMLET	Clothing and grooming	A band or bracelet worn round the upper part of a person's arm.	Person/Thing	
+3150	BOX OR CHEST	Modern world	Either a generic manmade object with space in it that is used to keep and store many various different objects, or a strong box with a lid on top.	Property	
+3151	CUT (WITH AXE)	Basic actions and technology	To slice or to divide into pieces with help of an axe.	Person/Thing	
```

Comment:
did you check for "chop"? Not there?

Reply:
Now I look up chop (def. : To cut meat, vegetables or fruits in very small pieces.)
But the cut (with axe) didn't indicate to cut into small pieces, so I add this new concept. 
I can also link cut with axe to chop.

Reply:
okay, link to "CHOP" (http://concepticon.clld.org/parameters/155). The links are bad there, so maybe make an issue that CHOP 155 and CHOP (IN PIECES) 2149 (http://concepticon.clld.org/parameters/2149) should be reconsidered, etc. But for the time being, this is our best guess.